src/atoms/progress-bar/progress-bar.tsx

```diff
@@ -21,6 +29,11 @@ interface ProgressBarProps {
     isErrored?: boolean;
     style?: ProgressBarStyles;
     value: number;
+
+    /**
+     * Unique identifier used to select the underlying <div> for functional/e2e testing
+     */
+    testId?: string;
```

Comment:
Nitpick - keep props alphabetical

Reply:
@brandongregoryscott Thanks for the feedback. I have updated the PR if you want to take a second look.